lib/doorkeeper/config/validations.rb

```diff
@@ -55,11 +55,17 @@ def validate_token_reuse_limit
       def validate_custom_access_token_attributes
         return if custom_access_token_attributes.blank?
 
+        # Need lib/doorkeeper#configure to set up the orm and initialize
+        # the access token and grant models before we can access them.
+        Doorkeeper.configuration
+
         custom_access_token_attributes.each do |attribute_name|
```

Comment:
Yeah it makes more sense to me to use `Doorkeeper.config.custom_access_token_attributes`

Also I'm wondering if we need this validation at all :thinking: We can use `model.column_names` and extract only those which are present, so if attribute isn't present in the model - it will be skipped.